### PR TITLE
Fix wizard css class

### PR DESCRIPTION
**Pull request by Toflar**
See #997

### CODE REVIEW

File: src/Resources/contao/classes/DataContainer.php [hunk 1]
```diff
@@ -526,19 +526,21 @@ protected function row($strPalette=null)
...
 			{
-				$arrData['eval']['tl_class'] .= ' wizard';
+				$cssClasses[] = 'wizard';
 			}
 		}
-		elseif (isset($arrData['eval']['tl_class']) && false !== strpos($arrData['eval']['tl_class'], 'wizard'))
+		elseif (in_array('wizard', $cssClasses))
 		{
-			$arrData['eval']['tl_class'] = str_replace('wizard', '', $arrData['eval']['tl_class']);
+			unset($cssClasses[array_search('wizard', $cssClasses)]);
```
Comment (leofeyer) [inline src/Resources/contao/classes/DataContainer.php]: We should handle the case that `array_search` returns `false` here.

**Comment by leofeyer** [comment]
I think this is the way to go. Thank you @Toflar.

File: src/Resources/contao/classes/DataContainer.php (hunk 1 above)
Comment (Toflar) [inline src/Resources/contao/classes/DataContainer.php]: Cannot happen because we check if it's present :-D

Comment (leofeyer) [inline src/Resources/contao/classes/DataContainer.php]: True.

**Comment by dmolineus** [comment]
This PR would fix the issue with css classes containing `wizard` but would still remove manually defined wizard classes.

**Comment by leofeyer** [comment]
I don't think so. It would only remove a class called `wizard` but not `wizard-2x` or `wizard-3x`.

**Pull request merged**